messages.go

```diff
@@ -37,34 +36,46 @@ const (
 	ResponseType ObliviousMessageType = 0x02
 )
 
+const ResponseSeedLength uint16 = 32
+
 type ObliviousDNSQuery struct {
-	ResponseKey []byte
-	DnsMessage  []byte
+	DnsMessage   []byte
+	ResponseSeed [ResponseSeedLength]byte
```

Comment:
Can we either revert this `ResponseSeed` change for now, or add the full key derivation bits? Right now, the seed is treated as a raw AES key, whereas it should be used to derive a key and nonce.